components/resources/BUILD.gn

```diff
@@ -89,11 +89,14 @@ repack("resources") {
     ]
   }
 
-  if (brave_wallet_enabled && !is_android) {
-    deps += [ "//brave/components/brave_wallet_ui:resources" ]
-    sources += [
-      "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak",
-    ]
+  if (brave_wallet_enabled) {
+    if (!is_android) {
+      deps += [ "//brave/components/brave_wallet_ui:resources" ]
+      sources += [ "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet.pak" ]
+    }
+
+    deps += [ "//brave/components/brave_wallet/resources:ethereum_provider_generated_resources" ]
+    sources += [ "$root_gen_dir/brave/components/brave_wallet/resources/brave_wallet_script_generated.pak" ]
```

Comment:
there's no reason to expose additional pak files here, we should repack everything inside brave_wallet into a single output. See https://github.com/brave/brave-core/pull/8703/files#diff-5303bb2b8878c4fce006fc3b8bb18f3d4e0f386dbea78d4c8c37a23323762389R55